config/internal/webhook/role_webhook.yaml.tmpl

```
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{.WebhookName}}
+  labels:
+    app: {{.WebhookName}}
+    component: data-science-pipelines
+rules:
+  - verbs:
+      - create
+      - get
+      - list
+      - watch
+      - update
+      - patch
+      - delete
+    apiGroups:
+      - pipelines.kubeflow.org
+    resources:
+      - pipelines
```

I think these permissions are also required. Could you please double check?
```suggestion
      - pipelines
      - pipelineversions
      - pipelineversions/finalizers
      - pipelineversions/status
```